frontend/compiled/term.go

```diff
@@ -175,3 +171,26 @@ func (t Term) string(sbb *strings.Builder, coeffs []big.Int) {
 	}
 	sbb.WriteString(strconv.Itoa(t.WireID()))
 }
+
+func (t Term[E, ptE]) HashCode() uint64 {
+	return t.Var
```

Comment:
@ivokub that seems too little? can we afford to do a better hashcode with the actual coeff limbs values ?

Reply:
never mind I recall now that we only care about sorting linear expressions with respect to variable id